Codigo/ModAreas.bas

```diff
@@ -353,7 +353,7 @@ Private Sub NotifyUser(ByVal UserNotificado As Integer, ByVal UserIngresante As
             If Not EsGM(UserNotificado) Or CompararPrivilegios(.flags.Privilegios, UserList(UserIngresante).flags.Privilegios) <= 0 Then
                 sendChar = False
             End If
-         ElseIf UserList(UserNotificado).flags.Muerto = 1 And MapInfo(.Pos.map).Seguro = 0 And (UserList(UserNotificado).GuildIndex = 0 Or UserList(UserNotificado).GuildIndex <> UserList(UserIngresante).GuildIndex Or modGuilds.NivelDeClan(UserList(UserIngresante).GuildIndex) < 6) Then
+         ElseIf UserList(UserNotificado).flags.Muerto = 1 And MapInfo(.pos.Map).Seguro = 0 And (UserList(UserNotificado).GuildIndex = 0 Or UserList(UserNotificado).GuildIndex <> UserList(UserIngresante).GuildIndex Or modGuilds.NivelDeClan(UserList(UserIngresante).GuildIndex) < 6) And Not IsSet(UserList(UserNotificado).flags.StatusMask, e_StatusMask.eTalkToDead) And Not IsSet(UserList(UserIngresante).flags.StatusMask, e_StatusMask.eTalkToDead) Then
```

Comment:
does it makes sense to check in the dead char has something equipped? dead character can't equip items

Reply:
true